src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.util.Limelight;
+import edu.wpi.first.math.geometry.Pose3d;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+public class Vision extends DashboardedSubsystem {
+
+    public static final String PHOTON_VISION_CAMERA_NAME = "photoncamera";
+
+    public static final int PIPELINE_CONE_INDEX = 0;
+    public static final int PIPELINE_CUBE_INDEX = 1;
+
+    public static final int PIPELINE_RETROREFLECTIVE_INDEX = 0;
```

Comment:
index 0 = high rrt (retro reflective tape)
index 1 = low rrt
index 2 = apriltag
also add a method that returns the tag id of the april tag (`return limelight.getID()` or something like that)